lib/publiccloud/utils.pm

```diff
@@ -193,9 +193,8 @@ sub is_hardened() {
 sub is_embargo_update {
     my ($incident, $type) = @_;
     return 0 if ($type =~ /PTF/);
-    script_retry("curl -sSf https://build.suse.de/attribs/SUSE:Maintenance:$incident -o /tmp/$incident.txt");
-    return 1 if (script_run("grep 'OBS:EmbargoDate' /tmp/$incident.txt") == 0);
-    return 0;
+    my $cmd = "curl -sSf https://smelt.suse.de/api/v1/basic/incidents/$incident/ | jq .embargo";
+    return index(script_output($cmd), 'null') != -1;
```

Comment:
LGTM, 
but what if:
```suggestion
    $cmd = script_output($cmd);
    return ($cmd =~ 'null');
```

Reply:
please check latest version

Reply:
Yes, suggestion applies also to latest: is_embargo when result is not null.
But your solution is ok too.